sources/packages/backend/apps/db-migrations/src/sql/Announcements/Rollback-create-announcements-table.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS sims.announcements;
```

Comment:
I think we are not using `IF EXISTS` anymore.

Reply:
Yes we have stopped using `IF EXISTS` and `IF NOT EXISTS`

Reply:
updated